torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -770,15 +770,21 @@ AOTITorchError aoti_torch_repeat_interleave_Tensor(
 }
 
 // Function to check existence of inf and NaN
-AOTITorchError aoti_check_inf_and_nan(AtenTensorHandle tensor) {
+AOTITorchError aoti_check_inf_and_nan(
```

Comment:
nit - this function looks very similar to another one (i.e. ```assert_inf_and_nan```) defined in utils.h. Could we only keep one?

Reply:
ditto on this one. `assert_inf_and_nan` seems only exists because of the non-ABI compatible mode. We can either ignore that mode or just call `assert_inf_and_nan` here.

Reply:
Resolved. Let the ABI-compat mode directly call into utils.